runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTile.tags.styles.ts

```diff
@@ -0,0 +1,96 @@
+import ExpandLessIcon from '@mui/icons-material/ExpandLess';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { Chip, Typography } from '@mui/material';
+import styled from 'styled-components';
+
+export const StyledChip = styled(Chip)`
+    margin: 0 8px 8px 0;
+`;
+
+export const StyledTypography = styled(Typography)(({ theme }) => `
```

Comment:
name it more meaningfully like e.g. "ChipName" or  "StyledChipName"

It doesn't have to be prefixed with Styled but ofc it makes it more readable